apiary.apib

```diff
@@ -1708,6 +1708,17 @@ Get a list of all phases a deal can go through, sorted by their order in the flo
             + (object)
                 + id: `21efc56e-1ba8-469d-926a-e89502591b47` (string)
                 + name: `New` (string)
+                + deal_status: `new` (enum[string])
```

Comment:
Could we get some more information why these fields are required?

During the initial deal API prototyping we discussed that we don't want new/open/won/lost to be part of deal phases, but as an additional status property on deal level. That way you can mark a deal as lost while it's in the "contact" deal phase or whatever.

Would it be possible to take this in scope while we're working on deals?

Reply:
The triggers are for creating particular actions when a deal enters a phase. In this case we want to show a pop-up for creating a meeting. This behaviour already exists in the old overview. 

We need the deal_status so that we can for example hide the lost deal phase or show the 15 most recent won deals.

Reply:
I think the consensus of our design meeting was that we can't change the way the phases work, because we can have phases after won or lost... so won & lost currently need to be phases... 
Yet, we still want to know which phases are for won, lost and new.
We call it "deal_status" because it describes the status of the deals within the phase, not of the phase themselves... If you have a better name suggestion, feel free to share...

Reply:
`related_deal_status`?